setup.py

```diff
@@ -550,6 +738,11 @@ def setup_paddle_extension() -> setuptools.Extension:
         "--expt-extended-lambda",
         "--use_fast_math",
     ]
+    if GLIBCXX_USECXX11_ABI:
+        # Core TE library was built with C++11 ABI (for PyTorch compat) so we need to do the same
+        # for all the framework extensions.
+        cxx_flags.append("-D_GLIBCXX_USE_CXX11_ABI=1")
+        nvcc_flags.append("-D_GLIBCXX_USE_CXX11_ABI=1")
```

Comment:
We should make this independent of the default value of `_GLIBCXX_USE_CXX11_ABI`:
```suggestion
    if GLIBCXX_USECXX11_ABI is not None:
        # Compile PaddlePaddle extensions with same ABI as core library
        flag = f"-D_GLIBCXX_USE_CXX11_ABI={int(GLIBCXX_USECXX11_ABI)}"
        cxx_flags.append(flag)
        nvcc_flags.append(flag)
```